cloudinit/config/cc_ca_certs.py

```diff
@@ -44,60 +44,110 @@
 from cloudinit import subp
 from cloudinit import util
 
-CA_CERT_PATH = "/usr/share/ca-certificates/"
-CA_CERT_FILENAME = "cloud-init-ca-certs.crt"
-CA_CERT_CONFIG = "/etc/ca-certificates.conf"
-CA_CERT_SYSTEM_PATH = "/etc/ssl/certs/"
-CA_CERT_FULL_PATH = os.path.join(CA_CERT_PATH, CA_CERT_FILENAME)
-
-distros = ['alpine', 'debian', 'ubuntu']
+DISTRO_DEFAULT_CONFIG = {
+    'ubuntu': {
+        'ca_cert_path': '/usr/share/ca-certificates/',
+        'ca_cert_filename': 'cloud-init-ca-certs.crt',
+        'ca_cert_config': '/etc/ca-certificates.conf',
+        'ca_cert_system_path': '/etc/ssl/certs/',
+        'ca_cert_update_exe': 'update-ca-certificates'
+    },
+    'debian': {
+        'ca_cert_path': '/usr/share/ca-certificates/',
+        'ca_cert_filename': 'cloud-init-ca-certs.crt',
+        'ca_cert_config': '/etc/ca-certificates.conf',
+        'ca_cert_system_path': '/etc/ssl/certs/',
+        'ca_cert_update_exe': 'update-ca-certificates'
+    },
+    'alpine': {
+        'ca_cert_path': '/usr/share/ca-certificates/',
+        'ca_cert_filename': 'cloud-init-ca-certs.crt',
+        'ca_cert_config': '/etc/ca-certificates.conf',
+        'ca_cert_system_path': '/etc/ssl/certs/',
+        'ca_cert_update_exe': 'update-ca-certificates'
+    },
+    'rhel': {
+        'ca_cert_path': '/usr/share/pki/ca-trust-source/',
+        'ca_cert_filename': 'anchors/cloud-init-ca-certs.crt',
+        'ca_cert_config': '',
```

Comment:
It's preferable to use `None` to indicate absence of a value, rather than the empty value.